_data/faq.yml

```diff
@@ -0,0 +1,25 @@
+- title: Quando potrò installare la app?
+  body: "Nell’estate 2018 inizieranno i primi test con i cittadini: questo significa che in una prima fase inviteremo un numero limitato di persone ad usare il servizio, e\
+        non sarà possibile per chiunque installare direttamente l’applicazione dagli store. Iscriviti alla newsletter per restare aggiornato. Il progetto è open source, se sei uno sviluppatore puoi seguirne gli sviluppi del progetto nei repositories \
+        [https://github.com/teamdigitale/italia-app](https://github.com/teamdigitale/italia-app) e [https://github.com/teamdigitale/italia-backend](https://github.com/teamdigitale/italia-backend)"
+- title: Che servizi saranno attivi quando inizierà la fase di test? 
+  body: "Questo dipende da dove risiedi. Molti servizi sono erogati dagli enti locali, quindi sperimenteremo la app in alcuni territori specifici, dove potremo integrare un numero significativo di servizi locali e nazionali"
+- title: Potrò usare tutti i servizi di io anche da browser web?
```

Comment:
Si era detto di usare una formattazione speciale per le occorrenze di _io_, si potrebbe usare semplicemente un bold per ora @matteodesanti?
Nota: ci sono altre occorrenze in questo file di seguito.